### PR TITLE
Disable mint sdk

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

**Comment by kumaryash90** [comment]
there seems to be an issue with latest update of foundry/forge, causing tests to fail (ref: https://github.com/foundry-rs/foundry/issues/2898)

File: contracts/base/ERC721Drop.sol [hunk 1]
```diff
@@ -96,8 +125,12 @@ contract ERC721Drop is ERC721SignatureMint, LazyMint, DelayedReveal, DropSingleP
          */
         address receiver = _req.to == address(0) ? msg.sender : _req.to;
 
+        if (_req.royaltyRecipient != address(0) && _req.royaltyBps != 0) {
+            _setupRoyaltyInfoForToken(tokenIdToMint, _req.royaltyRecipient, _req.royaltyBps);
+        }
+
         // Collect price
-        collectPriceOnClaim(_req.quantity, _req.currency, _req.pricePerToken);
+        collectPriceOnClaim(_req);
```
Comment (joaquim-verges) [inline contracts/base/ERC721Drop.sol:133]: this is the fix you were talking about the other day right?

**Review by joaquim-verges — APPROVED** [review]
lgtm

**Comment by joaquim-verges** [comment]
@kumaryash90 were you able to run the tests locally though?

**Pull request merged**